src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseSign1Message.cs

```diff
@@ -198,6 +308,16 @@ private static async Task<int> CreateCoseSign1MessageAsync(Stream content, byte[
             return writer.Encode(buffer);
         }
 
+        /// <summary>
+        /// Verifies that the signature is valid for the content using the specified key.
+        /// </summary>
+        /// <param name="key">The private key used to sign the content.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must match the value provided during signing.</param>
+        /// <returns><see langword="true"/> if the signature is valid; otherwise, <see langword="false"/>.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="key"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="key"/> is of an unsupported type.</exception>
+        /// <exception cref="InvalidOperationException">Content is detached from this message, use an overload that accepts a detached content.</exception>
+        /// <exception cref="CryptographicException"><see cref="CoseHeaderLabel.Algorithm"/> was missing, was incorrectly formatted, was not one of the supported values or doesn't match with the algorithms supported by the specified <paramref name="key"/>.</exception>
         public bool VerifyEmbedded(AsymmetricAlgorithm key, byte[]? associatedData = null)
```

Comment:
Each of these probably want a seealso to their Detached version (which wants one back to here)

Reply:
And possibly to the IsDetached member (or whatever we called it)

Reply:
> And possibly to the IsDetached member (or whatever we called it)

We don't have a public IsDetached, we do have `public ROM<byte>? Content`. I would prefer changing plaintext `Content` to a `<see cref="CoseMessage.Content"/>`.

Reply:
I think in the exception description it's more correct and natural to say "The content is detached from this message", and then just have a seealso link to the Content property.

If it's a link in the description it's not talking about the property, vs the concept, and the property can't be detached, so it'd be "The <see cref="Content" /> property indicates that the content is detached from this message."  If you prefer that over a seealso, I guess that's fine?